tests/core/embedders/CustomFontEmbedder.spec.ts

```diff
@@ -25,6 +25,16 @@ describe(`CustomFontEmbedder`, () => {
     expect(embedder.fontName).toBe('Ubuntu');
   });
 
+  it(`can set a custom font name`, async () => {
```

Comment:
In addition to this test, I think it would be good to create a simple [`PDFDocument` test](https://github.com/Hopding/pdf-lib/blob/master/tests/api/PDFDocument.spec.ts) that embeds a custom font with a `customName` and verifies that it serializes to the same value every time it's saved.